### DISCUSSION TITLE
This from scratch 2.0

### DISCUSSION THREAD

**Discussion by Scratcher3** [ScratchAddons/ScratchAddons#6888] [Ideas]
Where are the costume is change and you all in the costume tab It's like scratch 2.0 Where was the costume changes it will go to that costume in the costume tab

**Comment by mybearworld** [comment]
There's multiple ways to go about this:

1. Switch to the costume in the paint editor as well. This has the issue of being annoying when you want to edit one costume.
2. Deselect the costume currently focused in the paint editor, and focus the one used in the stage. This might make it difficult to tell which costume is applied. Also, I don't think there's a way to easily do that.
3. Select both the costume currently focused in the paint editor and the one used in the stage. I don't think there's an easy way to do that either.

**Reply by WorldLanguages** [reply]
Seems interesting, but I had no idea what OP meant. Could someone translate? 😅 Is this about a Scratch 2.0 behavior that was lost with 3.0?

**Reply by mxmou** [reply]
In 2.0, when the current sprite switched to a different costume, the new costume was automatically selected on the Costumes tab:
![costume_change_s2](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/7663904f-95e1-4ffb-b24f-b3d9c6f62e48)

**Reply by mybearworld** [reply]
But it's *not* actually switched to in the paint editor in 2.0 (e.g. number two), which can lead to situations like this:
![The Scratch 2.0 costume tab with the default two costumes of Scratch Cat, with costume2 being selected but costume1 visible in the paint eidot](https://github.com/ScratchAddons/ScratchAddons/assets/130385691/9d892e21-16c0-46c8-a9c2-e6923a438aaf)
